### PR TITLE
fix(assets): remove incorrect check for previously owned

### PR DESCRIPTION
**Pull request by whitneypurdum**
Screen Shot 2022-08-18 at 3 01 57 PM

### CODE REVIEW

File: src/app/routes/assets/asset-list/asset-list.component.html [hunk 1]
```diff
@@ -115,7 +115,7 @@
       </td>
     </ng-container>
 
-    <ng-container matColumnDef="owner" *ngIf="isPreviouslyOwned">
```
Comment (dawidgil) [inline src/app/routes/assets/asset-list/asset-list.component.html:118]: Yeah, it is wrong. But it should be `isOffered`.

Comment (whitneypurdum) [reply src/app/routes/assets/asset-list/asset-list.component.html:118]: Hmm I actually think it works without a check?

Comment (dawidgil) [reply src/app/routes/assets/asset-list/asset-list.component.html:118]: It works, but it is separation between those views in one component. Without that we do not know if this is for all tabs. later we might think if this is a bug, that this field is not displayed on the screen.

Comment (dawidgil) [reply src/app/routes/assets/asset-list/asset-list.component.html:118]: i would like in future to split this component into 3 components like in enrolments.

Comment (whitneypurdum) [reply src/app/routes/assets/asset-list/asset-list.component.html:118]: okay. There is already a check for the column types based on isOffered in the .ts file I think, which is why this column is not appearing for other asset views. But can add it.

Comment (dawidgil) [reply src/app/routes/assets/asset-list/asset-list.component.html:118]: if it is, then approved. Thanks for clarification and fixing.

**Review by dawidgil — APPROVED** [review]

**Pull request merged**